package.json

```diff
@@ -28,10 +30,11 @@
     "ts-loader": "^8.0.6"
   },
   "scripts": {
-    "build": "tsc",
+    "build": "tsc --outDir tmp && babel tmp --config-file ./config/babel.config.json --out-dir dist --copy-files && rm -rf tmp",
```

Comment:
👍 Compile Typescript to `/tmp` &rarr; transpile with babel from `/tmp` to `/dist` &rarr; remove `/tmp`.

Would it be worth adding a .gitignore for `/tmp`?

Reply:
Good shout.

Also, seeing you write it out like that, it occurred to me to split it into build steps (Make-style), so I've done that too. Let me know what you think!

Reply:
Funny you should say that, I originally wrote my comment that way, then thought maybe it’s good for other developers to also have to figure out things…

I think it’s much better DevX to have it in separate steps: easier to understand, debug and reuse!